_benchmark/commands/execute-test.md

```diff
@@ -81,78 +81,78 @@ After the test runs, OpenSearch Benchmark responds with a summary of the benchma
 
 ## Options
 
-Use the following options to customize the `execute-test` command for your use case. Options in this section are categorized by their use case.
+Use the following options to customize the `execute-test` command for your use case. Options in this section are categorized by their use case. 
 
 ## General settings
 
-The following options shape how each test runs and how results are output.
+The following options shape how each test runs and how results are output. 
```

Comment:
Is there a word we can use here other than "output"? I think it should technically be "outputted", but either one reads a bit awkwardly.

Reply:
Let's use "appear". I adjusted the suggestion accordingly.